redirects.yaml

```
@@ -354,6 +354,15 @@ redirects:
     - /docs/apps/manifest/...
   else: /docs/extensions/...
 
+- from: /docs/extensions/...
+  try:
+    - /docs/extensions/reference/...
+    - /docs/extensions/mv3/manifest/...
+    - /docs/extensions/mv3/...
```

I think we should also include MV2 docs in the fallback check. There are some pages that aren't in the MV3 doc set but are in the MV2 and apps one. In those cases, it would be best to load the "extensions" version of the page before falling back to the app docs.

```suggestion
    - /docs/extensions/mv3/...
    - /docs/extensions/mv2/manifest/...
    - /docs/extensions/mv2/...
```